bedita-app/controllers/api_base_controller.php

```diff
@@ -726,6 +732,44 @@ public function childrenFilter($conditions = null) {
     }
 
     /**
+     * Setup locale settings
+     * override FrontendController::setupLocale
+     * override AppController::setupLocale
+     *
+     * @see bedita-app/controllers/FrontendController#setupLocale()
+     * @see bedita-app/AppController#setupLocale()
+     */
+    protected function setupLocale() {
+        $lang = $this->lang;
```

Comment:
Do we need `$this->lang` ?
We could set `$this->currLang` just once, from query string making sure it exists in config: just see if `Configure::read('frontendLangs.' $lang)` exists